src/common/_modules/global/modules/dns/private_dns_zones.tf

```diff
@@ -59,6 +59,13 @@ resource "azurerm_private_dns_zone" "privatelink_servicebus" {
   tags = var.tags
 }
 
+resource "azurerm_private_dns_zone" "privatelink_servicebus_common" {
```

Comment:
This will lead to a failure the VNet already has a linked DNS zone with this name (lines 55-60). Unless that zone is moved to the common rg (as it should be), no other zones can be linked to the VNet